gst/nnstreamer/tensor_common.c

```diff
@@ -1235,14 +1267,16 @@ gst_gen_tensors_from_collectpad (GstCollectPads * collect,
     if (GST_IS_BUFFER (buf)) {
       mem = gst_buffer_get_memory (buf, 0);
       gst_buffer_append_memory (tensors_buf, mem);
+      printf ("Append Memory # %d (PTS : %lu)\n",
```

Comment:
change func to print logs (printf -> silent_debug ?)

Reply:
It is accidentally there. I will delete it. Thanks.